src/or/config.c

```diff
@@ -4567,14 +4562,28 @@ compute_real_max_mem_in_queues(const uint64_t val, int log_guess)
       result = ONE_GIGABYTE;
 #endif /* SIZEOF_VOID_P >= 8 */
     } else {
-      /* We detected it, so let's pick 3/4 of the total RAM as our limit. */
-      const uint64_t avail = (ram / 4) * 3;
+      /* We detected the amount of memory available. */
+      uint64_t avail = 0;
+
+      if (ram >= (8 * ONE_GIGABYTE)) {
+        /* If we have more than, or equal to, 8 GB of RAM available, we set the
+         * MaxMemInQueues to 0.4 * RAM, since the relay operator should
+         * probably be running multiple relays on this host.
```

Comment:
This was what the ticket was about: instead of running one Tor instance on a machine with a large amount of RAM and reserve 3/4 of it for our queues, we move to using 2/5 of the RAM and want people to run two relays. Maybe @teor2345 wants to elaborate here?